src/dodal/devices/zebra.py

```diff
@@ -223,7 +223,15 @@ def __str__(self) -> str:
         return ", ".join(input_strings)
 
 
+class SoftInputs(Device):
+    soft_in_1: EpicsSignal = Component(EpicsSignal, "SOFT_IN:B0")
+    soft_in_2: EpicsSignal = Component(EpicsSignal, "SOFT_IN:B1")
+    soft_in_3: EpicsSignal = Component(EpicsSignal, "SOFT_IN:B2")
+    soft_in_4: EpicsSignal = Component(EpicsSignal, "SOFT_IN:B3")
```

Comment:
Could: These are binary PVs, could we have an enum of something like `ON`/`OFF` to set them?